oap-server/server-core/src/main/java/org/apache/skywalking/oap/server/core/source/ScopeDefaultColumn.java

```diff
@@ -52,7 +52,7 @@ public ScopeDefaultColumn(String fieldName, String columnName, Class<?> type, bo
         /**
          * Dynamic active means this column is only activated through core setting explicitly.
          *
-         * @return
+         * @return false: Inactive dynamic column. true: Active dynamic column.
```

Comment:
This comment seems a little strange. 

```suggestion
         * @return FALSE: this column is not going to be added to the final generated metric as a column.
         *         TRUE: this column could be added as a column if core/activeExtraModelColumns == true.
```